docs/Filesystem-Paths.md

```diff
@@ -111,3 +112,66 @@ it may prove necessary during [[Troubleshooting]] or if you use
 or [Brimcap](https://github.com/brimdata/brimcap/blob/main/README.md) CLI tools
 to perform scripted operations from outside the app against data that's stored
 behind Brim.
+
+# Temporary Storage (all versions)
+
+Brim and the bundled Zed/Brimcap tools often make use of temporary storage.
+Some examples:
+
+* The **Zed** backend may use temporary storage to "spill to disk" when
+performing [`sort`](https://github.com/brimdata/zed/tree/main/docs/language/operators#sort),
+[`fuse`](https://github.com/brimdata/zed/tree/main/docs/language/operators#fuse),
+or [aggregations](https://github.com/brimdata/zed/blob/main/docs/language/aggregate-functions/README.md) on data sets that cannot fit into allocated system
+memory. For this, Zed uses a directory with a name that starts with
+`zed-spill-`. The directory is automatically deleted when the operation
+finishes.
+
+* **Brimcap** uses temporary directories to store the raw logs generated by
+[analyzers](https://github.com/brimdata/brimcap#included-analyzers) (e.g.,
+Zeek and Suricata) before they're loaded into a Pool in a Zed Lake. For this,
+Brimcap uses a directory with a name that starts with `brimcap-`. The
+directory is automatically deleted when `brimcap analyze` finishes.
+
+* In **Brim** when the **Packets** button is clicked to extract a flow from
+a larger pcap, the extracted packets are placed in a temporary file that's
+opened in Wireshark. For this, Brim uses a file with a name of the format
+`packets-[date+timestamp].pcap` (e.g., `packets-2020-03-09T17_27_53.399273Z.pcap`).
+This file is _not_ deleted automatically, as Brim does not make any assumptions
+about what you may continue to do with the pcap file outside the app.
+
+Standard conventions are used for creating these directories/files in the
+appropriate location for each supported platform. As a result, temporary
+storage should always be freed up automatically by the OS during normal
+operations, such as when performing a system restart. However, bugs or abrupt
```

Comment:
> As a result, temporary storage should always be freed up automatically by the OS during normal operations, such as when performing a system restart.

I don't think Windows does this.

Reply:
@nwt: Thanks for mentioning this! My bad... I'd checked on macOS and Ubuntu and made a bit of "QED" leap. Indeed, I just ran a quick test and confirmed that Windows does not clean `%TMP%` on a reboot. Then I was reminded of the many free (or paid!) "cleanup" tools for Windows that purport to address this pressing challenge. :roll\_eyes: 

As if that's not bad enough, this made me paranoid enough that I figured I couldn't necessarily assume that what I observed for Ubuntu could hold for other Linux distros, so I tried CentOS. I figured if I covered at least those two I could call it a trend. Sure enough, CentOS didn't clean `/tmp` between reboots either!

I've now pushed a commit that's more vague about how reboots or "common cleanup utilities" might reap the temp space, but no guarantees. I know I'm probably overcompensating by saying any of this, but since in light of recent bugs we know Zed has the capacity to chow a whole lot of temp space, I'd like to be able to say we'd warned the people _somewhere_...